internal/dataplane/parser/translate_ingress_test.go

```diff
@@ -273,7 +272,7 @@ func TestFromIngressV1beta1(t *testing.T) {
 		p := mustNewParser(t, store)
 
 		parsedInfo := p.ingressRulesFromIngressV1beta1()
-		assert.Equal(ingressRules{
+		assert.Equal(t, ingressRules{
```

Comment:
While you've already done the refactor I'm not pushing but instead of rewriting all the call sites you might consider changing the place of the `assert` object declaration: i.e. put it inside the subtest.

Reply:
If optimizing for the least changes as possible that would make sense, but I think it's better to simply not shadow the `assert` at all as it may become problematic. I don't mind passing `t` explicitly to every call, to be honest - it's just +3 characters and no additional lines. :)

Reply:
That's fine 👍